css/portal-dashboard/questions/multiple-choice-question.less

```diff
@@ -0,0 +1,27 @@
+@import "../variables";
+
+.choiceWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+  align-items: center;
+  margin-top: 12px;
+
+  .choiceIcon {
+    border-radius: 50%;
+    width: 20px;
+    height: 20px;
+    object-fit: contain;
+    border: 1.5px solid @cc-charcoal-light1;
+    background-color: white;
+    margin-left: 19px;
+    margin-right: 8px;
+  }
+  .correct {
+    font-weight: bold;
+  }
+}
+
+.questionText {
+  margin: 0px 20px 10px 20px;
+}
```

Comment:
newline

Reply:
Might still want to add a newline at the end of this file to be consistent.  I know we haven't enforced this as much in CSS modules, but seems like a good practice.